### PR TITLE
Create fix-non-callable-console-spam.rst

### PR DESCRIPTION
**Pull request by GanerCodes**

### CODE REVIEW

**Comment by GanerCodes** [comment]
I'm literally stupid at git, I guess merge both my pull requests and that adds the change and the news?

File: news/fix-non-callable-console-spam.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Fixed annoying console spam when calling a non-callable
```
Comment (jnoortheen) [inline news/fix-non-callable-console-spam.rst:19]: can it more descriptive without such strong words. Like, `hide warnings when autocompleting`

**Review by GanerCodes — COMMENTED** [review]
👍

**Comment by jnoortheen** [comment]
I have merged this to #4722

**Pull request closed without merging**